docs/docs/monitoring/analytics/data-structure-reference.mdx

```diff
@@ -781,6 +781,120 @@ Time of the last update of the slot in this session. The timestamp is a UTC.
 - Type: `DateTime`
 - Example: `2022-06-21 02:15:49.326936`
 
+## rasa_patterns
+
+Patterns are marker definitions that have been received from Rasa. This table is called `patterns`
+to distinguish them from extracted markers which are stored in `rasa_markers` table. It 
+stores the configuration of markers (which can be thought of as pattern
+of conversational events) along with their metadata.
+
+#### `id` pattern identifier
+
+The unique identifier of the rasa pattern is generated by Analytics.
+
+- Type: `varchar(36)`
+- Example: `bd074dc7-e745-4db6-86d0-75b0af7bc067`
+
+#### `name` pattern name
+
+Name of the pattern
+
+- Type: `varchar()`
+- Example: `registration success`
+
+#### `description` pattern description
+
+Description of the pattern
+
+- Type: `varchar()`
+- Example: `This marker identifies successful account registration in the chat`
+
+### `config` pattern configuration
+
+Pattern configuration dictionary stored as an escaped string
+
+- Type: `varchar()`
+- Example: `"{'or': [{'intent': 'mood_unhappy'},{'intent': 'mood_great'}]}"`
+
+#### `is_active` soft-delete flag
+
+Only patterns with `is_active==True` are processed during real-time analysis
+
+- Type: `boolean`
+
+#### `created_at` creation date time
+
+Time of creation of this pattern. The timestamp is a UTC.
+
+- Type: `DateTime`
+- Example: `2022-06-21 02:15:49.326936`
+
+#### `updated_at` update date time
+
+Time of the last update of the pattern in this session. The timestamp is a UTC.
+
+- Type: `DateTime`
+- Example: `2022-06-21 02:15:49.326936`
+
+## rasa_markers
+
+Markers are the extracted markers from the conversations. Each row in this table corresponds
+to a conversation where the marker was applied
```

Comment:
is it 'applied' or 'detected' ..? Not sure.. This sentence is a lil bit unclear to me. Does each row correspond to one conversation where the marker was detected?

Reply:
okay,  I don't have a strong opinion on this,

Reply:
The existing rasa documentation uses the collocation "markers applied" so I kept it that way, [here's the section](https://rasa.com/docs/rasa/markers/#extracted-markers).

> Does each row correspond to one conversation where the marker was detected?
Each row corresponds to one marker, not specifically to one conversation. I'll rephrase the sentence